.github/workflows/instruqt-track-deploy.yml

```diff
@@ -56,7 +56,7 @@ jobs:
     - name: Get Jobs State
       id: get-jobs-state
       run: |-
-        cat<<EOF >> job-state.json
+        cat<<"EOF" >> job-state.json
         ${{ toJSON(github) }}
         EOF
         cat job-state.json
```

Comment:
If we remove this line in the other scripts we should remove it in these.

Reply:
Good point, and I agree. I'll remove line 62 above (and the one noted below)